README.md

```diff
@@ -28,16 +28,16 @@
 
 ##<a name="section1"></a>What is Cygnus
 
-Cygnus is a connector in charge of persisting [Orion](https://github.com/telefonicaid/fiware-orion) context data in certain configured third-party storages, creating a historical view of such data. In other words, Orion only stores the last value regarding an entity's attribute, and if an older value is required then you will have to persist it in other storage, value by value, using Cygnus.

```

Comment:
I don't know why this edition has been included... there is no change on the text...